RELEASING.md

```diff
@@ -0,0 +1,43 @@
+# How to release like real pro
+
+Releasing a new version of Connaisseur includes the following steps:
+
+- adding a new version tag
+- creating a changelog from commit messages
+- creating a PR from `develop` (new version) to `master` (current version)
+- pushing a new version of the Connaisseur image to Dockerhub
+- merging in the PR
+- creating release page
+- shoot some trouble
+
+## Adding new Tag
+
+Before adding the new tag, make sure the Connaisseur version is updated in the `helm/values.yaml` and applies the semantic versioning guidelines: fixes increment PATCH version, non-breaking features increment MINOR version, breaking features increment MAJOR version. Then add the tag (on `develop` branch) with `git tag v<new-conny-version>` (e.g. `git tag v1.4.6`).
+
+## Creating Changelog
+
+The changelog can be automatically generated using the `scripts/changelogger.py` script. You have to update the two `ref1` and `ref2` variables in the main method with the the old and new git tags. So if you e.g. want to release a new version from `v1.4.5` to `v1.4.6`, then you have to set `ref1 = "v1.4.5"` and `ref2 = "v1.4.6"`. Then simply run `python scripts/changelogger.py > CHANGELOG.md`, storing the changelog in a new file `CHANGELOG.md`. This file will include all new commits, categorized by their type (e.g. fix, feat, docs, etc.), but may include some mistakes so take a manual look if everything looks in order. 
+
+This to look out for:
+
+- multiple headings for the same category
+- broken pull request links
+- `None` appended on end of line
+
+## Creating PR
+
+Create a PR from `develop` to `master`, putting the changelog a description and wait for someone to approve it.
+
+## Push new Connaisseur Image
+
+When the PR is approved and ready to be merge, first push the new Connaisseur image to Dockerhub, as it will be used in the release pipeline. Run `make docker` to build the new version of the docker image and then `DOCKER_CONTENT_TRUST=1 docker image push securesystemsengineering/connaisseur:<new-version>` to push and sign it. You'll obviously need the right private key and passphrase for doing so. You also need to be in the list of valid signers for connaisseur. If not already (you can check with `docker trust inspect securesystemsengineering/connaisseur --pretty`) you'll need to contact [Philipp Belitz](mailto:philipp.belitz@securesystems.de).
```

Comment:
```suggestion
When the PR is approved and ready to be merged, first push the new Connaisseur image to Dockerhub, as it will be used in the release pipeline. Run `make docker` to build the new version of the docker image and then `DOCKER_CONTENT_TRUST=1 docker image push securesystemsengineering/connaisseur:<new-version>` to push and sign it. You'll obviously need the right private key and passphrase for doing so. You also need to be in the list of valid signers for connaisseur. If not already (you can check with `docker trust inspect securesystemsengineering/connaisseur --pretty`) you'll need to contact [Philipp Belitz](mailto:philipp.belitz@securesystems.de).
```